scripts/midterm.py

```diff
@@ -0,0 +1,514 @@
+import sys
+
+import bin_response_by_predictor as brp
+import cat_correlation as cc
```

Comment:
This import throws an error because the "cat_correlation.py" file is not in your scripts folder, make sure to add this file then this import will work